scripts/run.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+
+# Install
+# set -euo pipefail
+
+# RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
```

Comment:
I'm not sure where the `rapids-*` stuff comes from. Probably defined in the image that set up above in `rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-25.02`? Probably uncomment this and try it out.

Reply:
I think they're from https://github.com/rapidsai/gha-tools/ (via https://docs.rapids.ai/resources/github-actions/).

Reply:
Yes, `gha-tools` provides these utilities. They are included in the RAPIDS CI images like `ci-conda` ([source](https://github.com/rapidsai/ci-imgs/blob/main/ci-conda.Dockerfile), [DockerHub](https://hub.docker.com/r/rapidsai/ci-conda)), and similarly for `ci-wheel` and `citestwheel`.

[`RAPIDS_CUDA_VERSION` and a few other env vars](https://github.com/rapidsai/ci-imgs/blob/8312f41003e8633a3117feca3fd90778f6d5a41e/ci-conda.Dockerfile#L101-L104) are set in those images as well.